routes/auth.js

```diff
@@ -1,7 +1,10 @@
 const express = require("express");
+const passport = require("passport");
 const router = express.Router();
 const auth = require("../controllers/auth");
 
+router.get("/github/login", passport.authenticate("github", { scope: ["user:email"] }));
```

Comment:
we can also keep a failure parameter here so that the user can know that login was not successful.

Something like this: https://github.com/jaredhanson/passport-github/blob/dee535a1d47ade9a15916a4f92522292b2e0ef46/README.md?plain=1#L104

Reply:
Thanks for the suggestion 🎉 , but if unsuccessful then it will give 401 Error
That has been handled here
https://github.com/Real-Dev-Squad/website-backend/blob/6ec922dd9cbff63045b43cab187d7b60d21e4bb3/controllers/auth.js#L18-L22